classes/recompletion_form.php

```diff
@@ -43,8 +46,8 @@ public function definition() {
         $mform->addElement('checkbox', 'enable', get_string('enablerecompletion', 'local_recompletion'));
         $mform->addHelpButton('enable', 'enablerecompletion', 'local_recompletion');
 
-        $options = array('optional' => false, 'defaultunit' => 86400);
-        $mform->addElement('duration', 'recompletionduration', get_string('recompletionrange', 'local_recompletion'), $options);
+        $options = array('optional' => false, 'defaultunit' => 'Y');
```

Comment:
it would be good for the default to stay the same as the current implementation "days" - in future it might be nice for this to have a site-configurable default option.